en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/getting-started/deploy/cc-as-a-standalone-gateway-on-kubernetes-helm-artifacts.md

```diff
@@ -0,0 +1,193 @@
+# Deploying Choreo Connect as a Standalone Gateway on Kubernetes - Helm Artifacts
+
+Let's deploy an API using WSO2 API Controller (apictl) on Choreo Connect, which is running on Kubernetes as a Standalone Gateway. You can select one of the option to install Choreo Connect Helm Chart in the middle of the documentation.
+
+- [Option 1: Install Chart from WSO2 Helm Chart Repository](#option-1-install-chart-from-wso2-helm-chart-repository)
+- [Option 2: Install Chart from Source](#option-2-install-chart-from-source)
+
+## Before you begin
+    
+-   WSO2 product Docker images used for the Kubernetes deployment.
+
+    WSO2 product Docker images available at [DockerHub](https://hub.docker.com/u/wso2/) package General Availability (GA)
```

Comment:
WSO2 product Docker images available at [DockerHub](https://hub.docker.com/u/wso2/) (General Availability (GA)
    versions of WSO2 products with no [WSO2 Updates](https://wso2.com/updates).)

Reply:
fixed with https://github.com/wso2/docs-apim/pull/5214/commits/56e8d06f0eefdc6ceb6870305d7af0214650c154